tests/test-cases/vendor-test.yaml

```diff
@@ -36,17 +37,11 @@ tests:
         - "./components/terraform/weather/main/providers.tf"
         - "./components/terraform/weather/main/variables.tf"
         - "./components/terraform/weather/main/versions.tf"
-        - "./components/terraform/myapp2/outputs.tf"
-        - "./components/terraform/myapp2/main.tf"
-        - "./components/terraform/myapp2/variables.tf"
-        - "./components/terraform/myapp2/versions.tf"
-        - "./components/terraform/myapp2/README.md"
-        # From ghcr.io/cloudposse/atmos/tests/fixtures/components/terraform/myapp:v0
-        - "./components/terraform/myapp1/outputs.tf"
-        - "./components/terraform/myapp1/main.tf"
-        - "./components/terraform/myapp1/variables.tf"
-        - "./components/terraform/myapp1/versions.tf"
-        - "./components/terraform/myapp1/README.md"
+        # From ghcr.io/cloudposse/atmos/tests/fixtures/components/terraform/mock:v0
```

Comment:
Why are we reducing the files we check from the OCI vendoring?

Reply:
because these files do not exist in the `mock` component. In the `mock` component we only have a `main.tf` with all inputs and outputs

Reply:
https://github.com/cloudposse/atmos/tree/chore/replace-mock-component/tests/fixtures/components/terraform/mock

Reply:
I was confused because I didn't see we also changed what the OCI contained.